packages/lesswrong/lib/routes.ts

```diff
@@ -304,12 +304,10 @@ if (forumTypeSetting.get() !== 'EAForum') {
       background: postBackground
     },
     {
-      name: 'tagIndex',
-      path: '/tags',
-      componentName: 'PostsSingleRoute',
-      _id:"DHJBEsi4XJDw2fRFq",
-      background: postBackground
-    },
+      name: "allTags",
```

Comment:
Route names should be unique and `allTags` is also used for the route that this redirects to.

(There should probably be a unit test for that, or maybe the name field should just not be here at all, since it doesn't do anything useful.)

Reply:
@jimrandomh I took out the duplicated name field. Good to go now?

Reply:
Hmm, I think you should give them a name, just not a duplicated one. At least all the other ones have names, and don't see a reason to break from that trend for now (and it might cause a bug).

Reply:
Okay, I put back in the original name which is not duplicated.